sci-mathematics/rstudio/rstudio-1.4.1106.ebuild

```diff
@@ -0,0 +1,217 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake java-pkg-2 java-ant-2 multiprocessing pam qmake-utils xdg-utils
+
+QT_VER=5.12.6
+QT_SLOT=5
+
+DESCRIPTION="IDE for the R language"
+HOMEPAGE="
+	https://rstudio.com
+	https://github.com/rstudio/rstudio/"
+SRC_URI="
+	https://github.com/rstudio/rstudio/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://s3.amazonaws.com/rstudio-dictionaries/core-dictionaries.zip -> ${P}-core-dictionaries.zip
+"
+
+LICENSE="AGPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="headless server test"
+REQUIRED_USE="headless? ( server )"
+# The test require an R package to be installed called `testthat`. I was able to
+# install this using R itself via:
+# ```sh
+# $ R
+# > install.packages("testthat")
+# // select a CRAN mirror
+# ```
+# Using this locally installed "testthat", I got the following results from the
+# test suite. It's unclear to me if the failures are from upstream or due to the
+# gentoo packaging. I don't really have the motivation to continue improving on
+# this, however I did want to leave this here for anyone who may be more
+# interested in the future.
+# [ FAIL 7 | WARN 6 | SKIP 2 | PASS 980 ]
+RESTRICT="test"
+
+RDEPEND="
+	dev-db/postgresql:*
+	dev-db/sqlite
+	dev-java/aopalliance:1
+	dev-java/gin:2.1
+	dev-java/javax-inject
+	=dev-java/validation-api-1.0*:1.0[source]
```

Comment:
is still this old version needed? could the version restriction be dropped and keep just the slot?

Reply:
@fordfrog  I'll grep around in the cmake files and see what I can find out.

Reply:
the easiest way to check is to drop the version restriction, emerge the newest `validation-api` and then try to emerge `rstudio` against it. if it compiles then the api used is compatible and the restriction is not needed anymore.

Reply:
> What's the outcome for this?

Have not tested yet